tests/devapps/DesktopTestApp/PublicClientHandler.cs

```diff
@@ -44,6 +45,7 @@ public async Task<AuthenticationResult> AcquireTokenInteractiveAsync(
             CreateOrUpdatePublicClientApp(InteractiveAuthority, ApplicationId);
 
             AuthenticationResult result;
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Maybe replace these with the new method and remove this line everywhere?

Reply:
I think this is the only file where I suppressed the warnings instead of making the actual change. Unlike all the other places, using the new API here would've meant multiple method signature changes, then changes to everything that referenced those methods, then changes to fields in different classes...

It was already started to be a big PR, and those suppressions seemed to be used in a lot of tests, so I figured I'd take a shortcut on this one test app.